File: lib/my_tab.dart
class MyTab {
  int id;
  String name;
  int order;

  MyTab(this.id, this.name, this.order);

  Map<String, dynamic> toMap() {
    return {
      'tab_id': id,
      'tab_name': name,
      'tab_order': order,
    };
  }
}

File: app/src/main/java/com/example/kifmatome/MyTab.kt
package com.example.kifmatome

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tab")
data class MyTab(
    @PrimaryKey(autoGenerate = true) val tid: Int,
    @ColumnInfo(name = "tab_name") val tabName: String?,
    @ColumnInfo(name = "tab_order") val tabOrder: Int
)
